resources/metadata/whitesource.yaml

```diff
@@ -398,109 +398,10 @@ spec:
     resources:
       - name: buildDescriptor
         type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: golang
       - name: opensourceConfiguration
         type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: golang
       - name: checkmarx
         type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: golang
-      - name: buildDescriptor
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: maven
-      - name: opensourceConfiguration
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: maven
-      - name: buildDescriptor
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: mta
-      - name: opensourceConfiguration
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: mta
-      - name: buildDescriptor
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: npm
-      - name: opensourceConfiguration
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: npm
-      - name: buildDescriptor
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: pip
-      - name: opensourceConfiguration
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: pip
-      - name: buildDescriptor
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: sbt
-      - name: opensourceConfiguration
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: sbt
-      - name: buildDescriptor
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: dub
-      - name: checkmarx
-        type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: dub
```

Comment:
Not obvious why all those lines are not needed anymore. I assume those are sub-sets of another stash?

Reply:
See answer in other comment.